### DISCUSSION TITLE
Incorrect payment received

### DISCUSSION THREAD

**Discussion by alisahinbalcioglu** [community/community#77928] [Copilot Conversations] [answered]
### Select Topic Area

General

### Body

I tried Copilot, but then I canceled it and it is not used. Every month, 38 dollars are charged to my card even though it is not used, I request that these charges be canceled and refunded.

**Comment by Akash1134** [comment, marked as answer]
Hi @alisahinbalcioglu ! The best place to go for answers to account-specific or billing related questions like this one is opening a ticket on our [Support page](https://support.github.com/contact). Please open a ticket with them :)
